libs/sdk-embedding/src/iframe/kd.ts

```diff
@@ -976,4 +986,34 @@ export namespace EmbeddedKpiDashboard {
         GdcKdEventType.DrillToUrlResolved,
         IDrillToUrlResolvedDataBody
     >;
+
+    /**
+     * Open schedule email dialog
```

Comment:
We should make it more descriptive (e.g. for what is it good for)?
And I'm also thinking about naming - does the command really schedule an email, or just open a dialog?

Reply:
This command only for opening the dialog schedule email